setup.py

```diff
@@ -52,10 +52,13 @@
 
 retrieve_chat_pgvector = [*retrieve_chat, "pgvector>=0.2.5"]
 
+psycopg = []
 if current_os in ["Windows", "Darwin"]:
-    retrieve_chat_pgvector.extend(["psycopg[binary]>=3.1.18"])
+    psycopg.extend(["psycopg[binary]>=3.1.18"])
 elif current_os == "Linux":
-    retrieve_chat_pgvector.extend(["psycopg>=3.1.18"])
+    psycopg.extend(["psycopg>=3.1.18"])
+
+retrieve_chat_pgvector.extend(psycopg)
```

Comment:
Why would this PR change the setting of `retrieve_chat_pgvector`? Is there an issue with current `retrieve_chat_pgvector`?